hooks/path_info.py

```diff
@@ -70,6 +80,7 @@ def get_publish_name(self, path, sequence=False):
 
         version_pattern_match = re.search(VERSION_REGEX, filename)
         frame_pattern_match = re.search(FRAME_REGEX, filename)
+        version_frame_pattern_match = re.search(VERSION_FRAME_REGEX, filename)
```

Comment:
Maybe we don't need the additional regex? Can we make this a 2-pass process if the additional arg is supplied? 

if the assumption is that the frame number is always after the version number (when both are included) maybe the logic should be:

```python
if strip_frame and frame_pattern_match:
    # remove the frame number with the existing frame regex

# continue with the existing logic...
```

Reply:
I think this is the best approach. I did consider that this might be possible, but in the end went with the simple option of formulating a specific regex.